go.mod

```diff
@@ -25,8 +25,28 @@ require (
 )
 
 require (
+	github.com/Azure/azure-event-hubs-go/v3 v3.3.16
+	golang.org/x/time v0.0.0-20210611083556-38a9dc6acbc6 // indirect
+	k8s.io/klog/v2 v2.9.0 // indirect
+	sigs.k8s.io/structured-merge-diff/v4 v4.1.2 // indirect
+)
+
+require (
```

Comment:
Will you run `go mod tidy`? Not clear why these are separate groups.

Reply:
I think this is a Go 1.17 thing. `make tidy` does not change anything on my end.